### PR TITLE
Polishing REST.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

File: Storage/test.cc [hunk 1]
```diff
@@ -914,11 +915,33 @@ TEST(TransactionalStorage, RESTfulAPIDoesNotExposeHiddenFieldsTest) {
...
 
-  EXPECT_TRUE(fields2.api.count("user") == 1);
-  EXPECT_TRUE(fields2.api.count("post") == 0);
-  EXPECT_EQ(1u, fields2.api.size());
+  EXPECT_TRUE(fields2.url_data.count("user") == 1);
+  EXPECT_TRUE(fields2.url_data.count("post") == 0);
+  EXPECT_EQ(1u, fields2.url_data.size());
+}
+
+TEST(TransactionalStorage, ShuttingDownAPIRepotsUpAsFalse) {
```
Comment (mzhurovich) [inline Storage/test.cc]: s/Repots/Reports/

File: Storage/api.h [hunk 2]
```diff
@@ -262,7 +262,7 @@ class RESTfulStorage {
         storage, restful_url_prefix, handlers_);
     // Register handlers on a specific port under a specific path prefix.
     for (const auto& handler : handlers_) {
-      const std::string route = path_prefix + '/' + handler.first;
+      const std::string route = path_prefix + "/data/" + handler.first;
```
Comment (mzhurovich) [inline Storage/api.h]: Make `"/data"` some named constant?

File: Storage/test.cc (hunk 1 above)
Comment (dkorolev) [inline Storage/test.cc]: :+1: https://github.com/dkorolev/Current/commit/30e9b509dc3aa1b7bddfa8b86f607f8280efbae5

File: Storage/api.h (hunk 2 above)
Comment (dkorolev) [inline Storage/api.h]: :+1: https://github.com/dkorolev/Current/commit/30e9b509dc3aa1b7bddfa8b86f607f8280efbae5

**Pull request merged**